dev/template.html

```diff
@@ -5,6 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=Edge">
     <title>Instant search demo built with instantsearch.js</title>
     <script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=default,Array.from"></script>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/instantsearch.css@7.0.0/themes/reset-min.css">
```

Comment:
The theme `algolia` already contains the `reset`. We don't need to include it.

Reply:
Oh good to know 👍